client/src/common/components/ListViewItem.jsx

```diff
@@ -8,6 +8,7 @@ import Preloader, { PreloaderSize } from 'common/components/Preloader';
 import { ListType } from 'modules/list/consts';
 import { ColorType, Routes } from 'common/constants/enums';
 import { IntlPropType } from 'common/constants/propTypes';
+import './ListViewItem.scss';
```

Comment:
Please split external and internal import groups.

Reply:
Here imports are split. `import { IntlPropType } from 'common/constants/propTypes';` is our internal import